### DISCUSSION TITLE
Application error: a client-side exception has occurred (see the browser console for more information)

### DISCUSSION THREAD

**Discussion by aditya305** [vercel/next.js#40629] [Help]
I deployed my project without **'_document.js'** website deploys but the moment, I add **'_document.js'**  show this error all though I'm using latest version of nextjs. 

Please help

https://flutter-mumbai-roadshow.vercel.app/

**Comment by hamzaaouni** [comment]
Hi @aditya305,
Verify the deployment URL once more. Verify the deployment's correct URL and that it corresponds to the deployment you are attempting to access. Verify whether the deployment is still in effect. To ensure the deployment is still in development and has not been deleted, check its status on your dashboard. Check to see if the deployment went well. Examine the deployment logs to see if there were any errors that might have contributed to the deployment failing. Check to see whether the server is having any problems. Server problems might occasionally result in failure or unavailability of deployments. To find out whether there are any known problems, check the server's status or get in touch with the support staff. Redeploying the application might work.
hope this helpful
